documentation/src/paradox/WSQuickStart.md

```diff
@@ -0,0 +1,119 @@
+# Quick Start to WS SSL
+
+This section is for people who need to connect to a remote web service
+over HTTPS, and don't want to read through the entire manual. If you
+need to set up a web service or configure client authentication, please
+proceed to the @ref:[next section](CertificateGeneration.md).
+
+## Connecting to a Remote Server over HTTPS
+
+If the remote server is using a certificate that is signed by a well
+known certificate authority, then WS should work out of the box without
+any additional configuration. You're done!
+
+If the web service is not using a well known certificate authority, then
+it is using either a private CA or a self-signed certificate. You can
+determine this easily by using curl:
+
+```
+curl https://financialcryptography.com # uses cacert.org as a CA
+```
+
+If you receive the following error:
+
+```
+curl: (60) SSL certificate problem: Invalid certificate chain
+More details here: http://curl.haxx.se/docs/sslcerts.html
+
+curl performs SSL certificate verification by default, using a "bundle"
+ of Certificate Authority (CA) public keys (CA certs). If the default
+ bundle file isn't adequate, you can specify an alternate file
+ using the --cacert option.
+If this HTTPS server uses a certificate signed by a CA represented in
+ the bundle, the certificate verification probably failed due to a
+ problem with the certificate (it might be expired, or the name might
+ not match the domain name in the URL).
+If you'd like to turn off curl's verification of the certificate, use
+ the -k (or --insecure) option.
+```
```

Comment:
Here as well text after `'` is highlighted differently.

![ssl_test](https://cloud.githubusercontent.com/assets/422086/20478078/4989a3ce-afe1-11e6-8911-1dc2445e02ec.png)